Doc/library/idle.rst

```diff
@@ -1,7 +1,9 @@
 .. _idle:
 
-IDLE
-====
+:mod:`idlelib` --- IDLE
+=======================
```

Comment:
IDLE is primarily an application, and secondly a module for use by IDLE extension writers.  So I wouldn’t change this heading!